okhttp-tests/src/test/java/okhttp3/CipherSuiteTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2016 Google Inc.
```

Comment:
Not yet. Things like that trigger new incentives and I'm not prepared to deal with the consequences.